composer.lock

```diff
@@ -3998,16 +3999,16 @@
         },
         {
             "name": "vimeo/psalm",
-            "version": "5.10.0",
+            "version": "5.15.0",
```

Comment:
Let's not upgrade anything: just the lock file.

Merging up is hell otherwise.

Reply:
Also, let's squash in the change that reverts all these lockfile changes :)